UGC build fails on amd64 after switch to llvm12

Hi, after version bumping and pulling your patches, changing to llvm12 and so on...  I am no longer able to update ungoogled-chromium. I think the issue is related to removing the https://github.com/flathub/org.chromium.Chromium/blob/3fc491081c2ef427ce18725518bf80c847214d60/patches/0022-Revert-Reland-build-In-LTO-links-run-just-one-link-a.patch patch. 

What doesn't make sense to me is that you don't have it either but don't seem affected. Do you have any idea what might be causing this for the UGC flatpak only? 

https://flathub.org/builds/#/builders/32/builds/47906 
```
[510/536] STAMP obj/build/config/shared_library_deps.stamp
[511/536] LINK ./nasm
FAILED: nasm 
python "../../build/toolchain/gcc_link_wrapper.py" --output="./nasm" -- clang++ -fuse-ld=lld -Wl,--fatal-warnings -Wl,--build-id=sha1 -fPIC -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,-z,defs -Wl,--as-needed -Wl,--icf=all -Wl,--color-diagnostics -Wl,-mllvm,-instcombine-lower-dbg-declare=0 -flto=thin -Wl,--thinlto-jobs=all -Wl,--thinlto-cache-dir=thinlto-cache -Wl,--thinlto-cache-policy,cache_size=10\%:cache_size_bytes=40g:cache_size_files=100000 -Wl,-mllvm,-import-instr-limit=5 -fwhole-program-vtables -Wl,--no-call-graph-profile-sort -m64 -Wl,-O2 -Wl,--gc-sections -Wl,--gdb-index -rdynamic -nostdlib++ -Wl,--lto-O0 -pie -Wl,--enable-new-dtags -L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed  -o "./nasm" -Wl,--start-group @"./nasm.rsp"  -Wl,--end-group  -ldl -lpthread -lrt
ld.lld: error: --thinlto-jobs=all: number expected, but got 'all'
ld.lld: error: --thinlto-jobs: number of threads must be > 0
clang-10: error: linker command failed with exit code 1 (use -v to see invocation)
[512/536] CC obj/third_party/ffmpeg/ffmpeg_internal/h264pred.o
ninja: build stopped: subcommand failed.
]2;flatpak-builder: Cleanup chromiumError: module chromium: Child process exited with code 1
FB: Unmounting read-only fs: fusermount -uz /srv/buildbot/worker/build-x86_64-2/build/.flatpak-builder/rofiles/rofiles-8w5KSI
```

TIA

I've just realized that you are affected too: https://flathub.org/builds/api/v2/logs/2021777/raw

```
$ curl -s https://flathub.org/builds/api/v2/logs/2021777/raw |grep -n -A10 '\[510\/536]'
19560:[510/536] STAMP obj/build/config/shared_library_deps.stamp
19561-[511/536] LINK ./nasm
19562-FAILED: nasm 
19563-python "../../build/toolchain/gcc_link_wrapper.py" --output="./nasm" -- clang++ -fuse-ld=lld -Wl,--fatal-warnings -Wl,--build-id=sha1 -fPIC -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,-z,defs -Wl,--as-needed -Wl,--icf=all -Wl,--color-diagnostics -Wl,-mllvm,-instcombine-lower-dbg-declare=0 -flto=thin -Wl,--thinlto-jobs=all -Wl,--thinlto-cache-dir=thinlto-cache -Wl,--thinlto-cache-policy,cache_size=10\%:cache_size_bytes=40g:cache_size_files=100000 -Wl,-mllvm,-import-instr-limit=5 -fwhole-program-vtables -Wl,--no-call-graph-profile-sort -m64 -Wl,-O2 -Wl,--gc-sections -Wl,--gdb-index -rdynamic -nostdlib++ -Wl,--lto-O0 -pie -Wl,--enable-new-dtags -L/app/lib -Wl,-z,relro,-z,now -Wl,--as-needed  -o "./nasm" -Wl,--start-group @"./nasm.rsp"  -Wl,--end-group  -ldl -lpthread -lrt
19564-ld.lld: error: --thinlto-jobs=all: number expected, but got 'all'
19565-ld.lld: error: --thinlto-jobs: number of threads must be > 0
19566-clang-10: error: linker command failed with exit code 1 (use -v to see invocation)
19567-[512/536] CC obj/third_party/ffmpeg/ffmpeg_internal/h264pred.o
19568-ninja: build stopped: subcommand failed.
19569-ninja: Entering directory `out/Release'
19570-[1/49180] STAMP obj/base/numerics/base_numerics.stamp
```

These are errors from your log as well. I'll fix it by enabling llvm12 before building ffmpeg

--- a/build-aux/bootstrap.sh
+++ b/build-aux/bootstrap.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
 
 # DO NOT REUSE THE BELOW API KEYS; these are for Flathub only.
 # http://lists.debian.org/debian-legal/2013/11/msg00006.html

--- a/build-aux/build.sh
+++ b/build-aux/build.sh
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-ninja -C out/ReleaseFree -j$FLATPAK_BUILDER_N_JOBS libffmpeg.so
-
 . /usr/lib/sdk/node12/enable.sh
 . /usr/lib/sdk/openjdk11/enable.sh
 . /usr/lib/sdk/llvm12/enable.sh
+ninja -C out/ReleaseFree -j$FLATPAK_BUILDER_N_JOBS libffmpeg.so
 ninja -C out/Release -j$FLATPAK_BUILDER_N_JOBS chrome

--- a/build-aux/build.sh
+++ b/build-aux/build.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
 
 . /usr/lib/sdk/node12/enable.sh
 . /usr/lib/sdk/openjdk11/enable.sh

--- a/build-aux/install.sh
+++ b/build-aux/install.sh
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
 
 mkdir -p /app/chromium
 

